### PR TITLE
bug 1803193: Tighten security groups

### PR DESCRIPTION
**Pull request by MaysaMacedo**
This commit ensures the security group are limiting access
to the minimum required.

### CODE REVIEW

**Comment by MaysaMacedo** [comment]
Still needs to be verified.

File: pkg/platform/openstack/kuryr_bootstrap.go [hunk 1]
```diff
@@ -1023,16 +1023,12 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 		return nil, errors.Wrap(err, "failed to add rule opening traffic pod to pod")
 	}
 	for _, cidr := range podSubnetCidrs {
-		err = ensureOpenStackSgRule(client, masterSgId, cidr, -1, -1)
-		if err != nil {
-			return nil, errors.Wrapf(err, "failed to add rule opening traffic to masters on %s", cidr)
-		}
 		err = ensureOpenStackSgRule(client, workerSgId, cidr, -1, -1)
```
Comment (dulek) [inline pkg/platform/openstack/kuryr_bootstrap.go]: Why would the traffic to the workers be still open?

Comment (MaysaMacedo) [reply pkg/platform/openstack/kuryr_bootstrap.go]: The console pods require traffic to be open on the worker, and the etcd pods required traffic to be open on the master.

**Comment by dulek** [comment]
/lgtm

This seems okay, but I wonder if we aren't blocking some feature that is not required for bare install. Had you tried running conformance tests on that?

**Comment by MaysaMacedo** [comment]
> /lgtm
> 
> This seems okay, but I wonder if we aren't blocking some feature that is not required for bare install. Had you tried running conformance tests on that?

@dulek I did run conformance on osp13 both with and without the changes, the number of failing tests was almost the same.  I also testes in osp16 with the changes, I'm testing without now to compare.

**Pull request merged**